optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -201,6 +210,10 @@ public boolean isSuccess() {
         return succeeded != null && succeeded.booleanValue();
     }
 
+    public boolean isInitialized() {
+        return initialized != null && initialized.booleanValue();
```

Comment:
There's common-lang's BooleanUtils.isTrue(initialized) that does the same thing (null safe)

But I think this method  boolean isInitialized() is not desired:
I think it's better to use Boolean getInitialized() WITHOUT a null check (neither here nor in the calling code). Why? Because we can guarantee that setInitialized() has been called with a non-null value before getInitialized() is used. If that is not the case, that's a bug and I 'd rather see a NullPointerException than it silently presuming its not initialized, which might be incorrect (which might lead users to make big wrong decisions, like throw away an actually better solver configuration).

This is the principle of Fail Fast: if the state is corrupted, don't sweep it under the carpet, but throw any exception immediately, so the user knows there's something broken and he can report it to us. In the long term, this means we spot bugs sooner, which means less bugs and people thrust our software more. Thrust means not giving incorrect information (even if that means stating that something went wrong and it's unable to deliver the information).